api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -207,6 +215,14 @@ public Profile getProfile(DbUser user) {
     profile.setFreeTierUsage(freeTierBillingService.getUserCachedFreeTierUsage(user));
     profile.setFreeTierDollarQuota(freeTierBillingService.getUserFreeTierDollarLimit(user));
 
+    Optional<DbUserTermsOfService> latestTermsOfServiceMaybe =
+        userTermsOfServiceDao.findFirstByUserIdOrderByTosVersionDesc(user.getUserId());
+    if (latestTermsOfServiceMaybe.isPresent()) {
```

Comment:
nit: one pattern that's kinda nice is to do:
```
latestTermsOfServiceMaybe.ifPresent(profile::setTermsOfServiceFields)
```
where `setTermsOfServiceFields` is a helper method that sets both these things.